components/navigation.tsx

```
@@ -189,9 +154,7 @@ export default function Navigation() {
         </nav>
       </div>
       <div className="hidden md:flex items-center gap-4">
-        <div className="relative w-full max-w-sm">
-          <SearchBar />
-        </div>
+        <div className="relative w-full max-w-sm"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)